crates/aranya-daemon-api/src/lib.rs

```diff
@@ -1,3 +1,14 @@
+//! The Aranya Daemon API.
+//!
+//! The Aranya Daemon API is a shared interface between the Aranya client and
+//! daemon. This crate mainly contains a trait that defines the RPC calls that
+//! [`tarpc`] uses to communicate and also handles type conversions
+//! between the external client and internal Aranya functionality.
+//!
+//! [`tarpc`]: https://crates.io/crates/tarpc
+//!
+//! For more information, refer to: <https://docs.rs/aranya-client/latest/aranya_client/>
```

Comment:
```suggestion
//! For more information, refer to: <https://docs.rs/aranya-client>
```
works as a shorter redirect to "latest"